include/mbedtls/check_config.h

```diff
@@ -962,7 +962,9 @@
 #error "MBEDTLS_SSL_VARIABLE_BUFFER_LENGTH defined, but not all prerequisites"
 #endif
 
-
+#if defined(MBEDTLS_SSL_CONTEXT_SERIALIZATION) && !( defined(MBEDTLS_GCM_C) || defined(MBEDTLS_CCM_C) || defined(MBEDTLS_CHACHAPOLY_C) )
```

Comment:
This should also be documented in `mbedtls_config.h`. As a general rule, we need to remember to keep those two files in sync.